src/components/vm/snapshots/vmSnapshotsCard.jsx

```diff
@@ -16,17 +16,20 @@
  * You should have received a copy of the GNU Lesser General Public License
  * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
  */
+
 import React from 'react';
 
-import cockpit from 'cockpit';
```

Comment:
This can be very nicely enforced and autofixed globally as well. I do it in anaconda-webui, see https://github.com/rhinstaller/anaconda-webui/blob/main/.eslintrc.json#L32